packages/core/test/shared/helpers.ts

```diff
@@ -1,7 +1,106 @@
+import { getDefaultLoggerOptions, pino } from "@walletconnect/logger";
+import { CoreTypes, ICore } from "@walletconnect/types";
+import { DEFAULT_DB_NAME, MOCK_STORE_NAME, TEST_CORE_OPTIONS, storeTestValues } from "./values";
+import { Core, Store } from "../../src";
+
+export type coreStorageData = {
+  storage?: any;
+  histroy?: any;
+  expirer?: any;
+  keychain?: any;
+  pairing?: any;
+  session?: any;
+};
+export type MockValue = { id: string; value: string };
```

Comment:
Makes it clearer that this is specifically a value for `Store`

```suggestion
export type MockStoreValue = { id: string; value: string };
```